owtf/settings.py

```diff
@@ -215,3 +201,8 @@
         exec(settings, globals(), locals())
 except FileNotFoundError:
     pass
+
+# JWT
+JWT_SECRET_KEY = "epb$9n04zst7l%wr9iw$cugk2*_^_aud5=snyn1^k*b*)ab@^i"
```

Comment:
add a default one -> something like "changeme" and add a comment to add a new secret key after installation?

Reply:
![image](https://user-images.githubusercontent.com/37950602/121945801-ced50e80-cd71-11eb-94f7-5d91aa8f77c6.png)
After installation, showing this message to change the default JWT_SECRET_KEY in settings.py